docs/integrations/sources/cart.md

```diff
@@ -51,4 +51,6 @@ Please follow these [steps](https://developers.cart.com/docs/rest-api/docs/READM
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.2   | 2021-08-17 | [5465](https://github.com/airbytehq/airbyte/pull/5465) | Fixing of CI tests|
```

Comment:
better description: added end_date config option to limit the amount of synced data

Reply:
added